pkg/features/kube_features.go

```diff
@@ -507,6 +513,7 @@ var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureS
 	TTLAfterFinished:                            {Default: false, PreRelease: utilfeature.Alpha},
 	KubeletPodResources:                         {Default: false, PreRelease: utilfeature.Alpha},
 	WindowsGMSA:                                 {Default: false, PreRelease: utilfeature.Alpha},
+	NonPreemptingPriority:                       {Default: false, PreRelease: utilfeature.Alpha},
```

Comment:
I wonder if we can enable this by default. With defaulting, upgrades should work fine.
@liggitt